src/styles/blocks/contentText.scss

```diff
@@ -0,0 +1,15 @@
+.contentText {
+  margin-bottom: 4px;
+  color: $main-text-color;
+  width: 166px;
+  height: 36px;
+  top: 207px;
+  left: 17px;
+  font-family: $main-font-famaly;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Ensure that the variable is correctly defined and imported.